src/AccordionItem/accordionItem.module.scss

```diff
@@ -11,6 +11,38 @@
   opacity: 0.9;
 }
 
+// this styling is applied when there is only 1 item in the accordion; MM designs
+.singleItemAccordion {
+  border: 1px solid #f1f1f2;
+  box-sizing: border-box;
+  box-shadow: 0px 0px 0px rgba(0, 0, 0, 0.5);
+  border-radius: 2px;
+  margin-bottom: 10px !important;
+
+  .accordionItem {
+    border-bottom: none;
+    padding: 0;
+    margin: 0;
+
+    span {
```

Comment:
my preference is that these are classNames rather than specific elements, but I'm not sure of the use case

Reply:
ah it's because i ported them over from MM and that's how i could get them most specific - ill change to teh class names now that they're accessible to me.